Test/Graph/graph_input_tests.hpp

```diff
@@ -23,7 +29,7 @@ std::vector<netket::json> GetGraphInputs() {
   // Hypercube 3d
   pars = {
       {"Graph",
-       {{"Name", "Hypercube"}, {"L", 20}, {"Dimension", 3}, {"Pbc", true}}}};
+       {{"Name", "Hypercube"}, {"L", 10}, {"Dimension", 3}, {"Pbc", true}}}};
```

Comment:
what was wrong with 20? Too slow to perform the BFS? (It shouldn't I guess)

Reply:
Well, with optimizations it was resonably fast, it just took some time in debug mode. But this probably shouldn't be left in for the PR. I can remove it.

Reply:
It's OK, I was just wondering about production! Just leave it to 10, it doesn't make a difference anyway for testing

Reply:
Alright. Also note that for `N=20` in the test BFS started once for every node and then visits every node once which means `8000 * 8000 = 64e6` calls to the `visitor_function` in that test case.